workshop1/03-connecting-the-api.md

```diff
@@ -135,6 +136,12 @@ as application state, and is fetched when the page first loads.
 3. Update the template returned by your app component to reference the `pets`
    state variable instead of `mockedPets`.
 
+4. Navigate to the node terminal, within Visual Studio Code. You will notice a yellow message saying your code was `Compiled with warnings`. Because we've changes a few things during this step, there are some variables we are not using anymore: `logo` and `mockedPets`.  
+Even though they don't have any effect on the code, it's good practice to keep your code as clean as possible, and it includes making sure you don't have any unused code (which will clutter your project as it accumulates, and don't bring any value).  
```

Comment:
Tiny visual thing, but could you add extra line-breaks here so that these appear as separate paragraphs within the bullet point? e.g.:

```markdown
1. Here's a paragraph...

   And here's a second.

   _Always keep an eye on..._
```